src/services/codefixes/parameterPropertyToDeclaration.ts

```diff
@@ -0,0 +1,90 @@
+/* @internal */
+namespace ts.codefix {
+    interface ParameterPropertyToPropertyInfo {
+        parameter: ParameterPropertyDeclaration
+        ctor: ConstructorDeclaration
+        container: ClassLikeDeclaration
+    }
+    export function rangeContainsSkipTrivia(r1: TextRange, node: Node, file: SourceFile): boolean {

```

Comment:
This looks like a fairly specialized function with only one caller.  Personally, I would inline it.  Otherwise, it might make sense to drop the `export`?